salt/metalk8s/roles/node/init.sls

```diff
@@ -1,3 +1,7 @@
 include:
-  - metalk8s.roles.internal.node-without-calico
+  - metalk8s.node.grains
+  - metalk8s.kubernetes.cni.loopback
   - metalk8s.kubernetes.cni.calico
+  - metalk8s.kubernetes.kubelet
+  - metalk8s.kubernetes.apiserver-proxy
+  - metalk8s.internal.preflight
```

Comment:
Why the re-ordering, compared to what was done before?

Reply:
No real reason, just to have both cni stuff grouped, since the ordering here does not seem to be important.

Reply:
Mmh, OK. And do we still need to include `cni.loopback` before `cni.calico`?

Reply:
Do you mean, do we still need to include it or do we need to include it "before" ?

Reply:
I've completely removed the loopback plugin.